packages/ra-auth-auth0/Readme.md

```diff
@@ -0,0 +1,100 @@
+# ra-auth-auth0
+
+An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Auth0](https://auth0.com) instance.
+
+This package provides:
+
+-   An `Auth0AuthProvider` for react-admin
+
+## Installation
+
+```sh
+yarn add ra-auth-auth0
+# or
+npm install --save ra-auth-auth0
+```
+
+## Example usage
+
+```jsx
+// in src/App.tsx
+import React, { useEffect, useRef, useState } from 'react';
+import { Admin, Resource } from 'react-admin';
+import { Auth0AuthProvider, httpClient } from 'ra-auth-auth0';
```

Comment:
`httpClient` is not used in this example

Reply:
```suggestion
import { Auth0AuthProvider } from 'ra-auth-auth0';
```